crypto/rsa/rsa_pmeth.c

```diff
@@ -358,17 +360,29 @@ static int pkey_rsa_decrypt(EVP_PKEY_CTX *ctx,
      */
     RSA *rsa = (RSA *)EVP_PKEY_get0_RSA(ctx->pkey);
 
-    if (rctx->pad_mode == RSA_PKCS1_OAEP_PADDING) {
+    if (rctx->pad_mode == RSA_PKCS1_OAEP_PADDING
+         || rctx->pad_mode == RSA_PKCS1_WITH_TLS_PADDING) {
```

Comment:
indentation looks awry here.  Indentation is always by 4 spaces (or optionally eight in this case).

Reply:
Thanks, addressed.